source/jaster/cli/core.d

```diff
@@ -421,7 +432,7 @@ final class CommandLineInterface(Modules...)
                 else if(this._defaultCommand == CommandInfo.init)
                 {
                     parseResult.type      = ParseResultType.commandNotFound;
-                    parseResult.helpText ~= format("ERROR: Unknown command '%s'.\n\n", parseResult.argParserBeforeAttempt.front.value);
+                    parseResult.helpText ~= this.makeErrorf("Unknown command '%s'.\n", parseResult.argParserBeforeAttempt.front.value);
                     parseResult.helpText ~= this.createAvailableCommandsHelpText(parseResult.argParserBeforeAttempt).toString();
```

Comment:
I believe that all help texts can be compile-time constants or be printed without memory allocation.

Reply:
Fortunately the minor amount of allocations we're doing here isn't worth stressing about. Going with `.format` and `.makeErrorf` everywhere isn't needed, but it's just convenient.

Reply:
Yeah, this is not something that should be worried about.